tests/checks/test_python.py

```diff
@@ -239,6 +277,139 @@ def test_missing_required_use(self):
                                  'python_targets_python3_6 '
                                  'python_single_target_python2_7 '
                                  'python_single_target_python3_6',
+                            RDEPEND='python_single_target_python2_7? ( '
+                                    '  dev-lang/python:2.7 ) '
+                                    'python_single_target_python3_6? ( '
+                                    '  dev-lang/python:3.6 )',
                             REQUIRED_USE='|| ( python_targets_python2_7 '
                                          '  python_targets_python3_6 )')),
             python.PythonMissingRequiredUSE)
+
+    def test_missing_deps(self):
```

Comment:
Choosing this as a random place to comment on all the tests, it would be nice to add some additional comments between the various assertions (or groups of similar assertions) that describe what makes them different then the other ones under the same method if applicable.

Alternatively, you could also add error messages to be output on test failure where it makes sense.

Reply:
Added some comments where tests were non-obvious. Figured it doesn't make sense to say that test for missing PYTHON_DEPS with entirely missing RDEPEND needs special description ;-). Please let me know if you find any of the remaining tests confusing.